### ISSUE TITLE
Rake task missing

### ISSUE THREAD

**Issue by noctivityinc** [Effilab/bing_ads_ruby_sdk#51]
After installing the gem the rake task is not found in the list of available tasks.

```
rake bing_token:get[bing_token.json,1070T4E69T912273,687249]
rake aborted!
Don't know how to build task 'bing_token:get' (See the list of available tasks with `rake --tasks`)
```

**Comment by danielmbarlow** [comment]
@noctivityinc

I just did a search on the codebase and didn't find any reference to `bing_token:get` in documentation or the git logs. If you see some reference to that somewhere in the project, could you post a link or a screen capture, please?

On the other hand there are these tasks:

```ruby
$ bin/rake --tasks | grep token
rake token:load_from_json[mcc,json_filepath]                                # Save authentication data to database
rake token:update[mcc]                                                      # Update test config file and store with Bing OAuth2 token
```

**Comment by noctivityinc** [comment]
It’s literally the first step in your read me.

**Comment by danielmbarlow** [comment]
:laughing: You're right, thanks. We'll update the doc.

Does `token:load_from_json` do what you need?

**Comment by danielmbarlow** [comment]
Sorry for my misleading answer. I believe that #55 should fix the issue. I have tested that the Rake task appears in local development. @noctivityinc, are you using the SDK in a Rails app? Would you mind trying out that version to see if you can get OAuth credentials, please? Please post any feedback on the PR.

**Comment by danielmbarlow** [comment]
#71 appears to be a duplicate of this issue

**Comment by danielmbarlow** [comment]
We're preparing a release for this bug fix in #72

**Comment by danielmbarlow** [comment]
Meanwhile, feel free to use the latest version of `master` whilst the release is being prepared:
```
# Gemfile
gem "bing_ads_ruby_sdk", github: "Effilab/bing_ads_ruby_sdk", branch: "master"
```

(I have noticed a problem in our build process, so we have to fix that before releasing v1.5.1)

**Comment by danielmbarlow** [comment]
Release [v.1.5.2](https://rubygems.org/gems/bing_ads_ruby_sdk/versions/1.5.2), so I'm closing this for now. Feel free to re-open if that doesn't resolve the problem.
